cdn/rpcserver/rpcserver.go

```diff
@@ -103,6 +103,10 @@ func (css *Server) ObtainSeeds(ctx context.Context, req *cdnsystem.SeedRequest,
 		span.RecordError(err)
 		return err
 	}
+	//css.service.AddTaskGCSubscriber(req.TaskId, &task.GCSubscriberInstance{
```

Comment:
remove

Reply:
PR has not been completed, now it is just testing, and the comments will be deleted later, for the same reason above